### PR TITLE
JAVA-15621

### PR DESCRIPTION
**Pull request by jsgrah-spring**

### CODE REVIEW

File: spring-reactive-modules/spring-5-reactive/src/main/java/com/baeldung/websocket/WebSocketController.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import java.util.concurrent.ConcurrentMap;
+
+@ServerEndpoint("/event-emitter")
+public class WebSocketController {
+
+    @OnOpen
+    public void onOpen(Session session, EndpointConfig endpointConfig) throws IOException {
+        // Get session and WebSocket connection
+        session.setMaxIdleTimeout(0);
+        System.out.println("Get session and WebSocket connection");
```
Comment (andrzejsydor) [inline spring-reactive-modules/spring-5-reactive/src/main/java/com/baeldung/websocket/WebSocketController.java:16]: why not to use some logger?

File: spring-reactive-modules/spring-5-reactive/src/main/java/com/baeldung/websocket/ReactiveWebSocketConfiguration.java [hunk 2]
```diff
@@ -32,6 +41,25 @@ public HandlerMapping webSocketHandlerMapping() {
...
+    }
+
+    @Bean
+    public WebSocketService webSocketService() {
+        TomcatRequestUpgradeStrategy tomcatRequestUpgradeStrategy = new TomcatRequestUpgradeStrategy();
+        tomcatRequestUpgradeStrategy.setMaxSessionIdleTimeout(10000L);
+        tomcatRequestUpgradeStrategy.setAsyncSendTimeout(10000L);
+        return new HandshakeWebSocketService(tomcatRequestUpgradeStrategy);
+    }
+    @Bean
```
Comment (andrzejsydor) [inline spring-reactive-modules/spring-5-reactive/src/main/java/com/baeldung/websocket/ReactiveWebSocketConfiguration.java:54]: should be empty line added

File: spring-reactive-modules/spring-5-reactive/src/main/java/com/baeldung/websocket/ReactiveWebSocketConfiguration.java [hunk 3]
```diff
@@ -3,26 +3,35 @@
...
+import org.springframework.context.annotation.Primary;
 
 @Configuration
+@EnableWebSocket
 public class ReactiveWebSocketConfiguration {
 
     @Autowired
     @Qualifier("ReactiveWebSocketHandler")
-    private WebSocketHandler webSocketHandler;
+    ReactiveWebSocketHandler reactiveWebSocketHandler;
```
Comment (andrzejsydor) [inline spring-reactive-modules/spring-5-reactive/src/main/java/com/baeldung/websocket/ReactiveWebSocketConfiguration.java:29]: why you delete the `private` ?

**Review by andrzejsydor — CHANGES_REQUESTED** [review]

File: spring-reactive-modules/spring-5-reactive/src/main/java/com/baeldung/websocket/WebSocketController.java (hunk 1 above)
Comment (jsgrah-spring) [reply spring-reactive-modules/spring-5-reactive/src/main/java/com/baeldung/websocket/WebSocketController.java:16]: I have updated to use logger.

File: spring-reactive-modules/spring-5-reactive/src/main/java/com/baeldung/websocket/ReactiveWebSocketConfiguration.java (hunk 2 above)
Comment (jsgrah-spring) [reply spring-reactive-modules/spring-5-reactive/src/main/java/com/baeldung/websocket/ReactiveWebSocketConfiguration.java:54]: I do not understand well what is ment with the empty line added here.

File: spring-reactive-modules/spring-5-reactive/src/main/java/com/baeldung/websocket/ReactiveWebSocketConfiguration.java (hunk 3 above)
Comment (jsgrah-spring) [reply spring-reactive-modules/spring-5-reactive/src/main/java/com/baeldung/websocket/ReactiveWebSocketConfiguration.java:29]: I have restored the privacy of this injected object.

File: spring-reactive-modules/spring-5-reactive/src/main/java/com/baeldung/websocket/WebSocketController.java (hunk 1 above)
Comment (andrzejsydor) [reply spring-reactive-modules/spring-5-reactive/src/main/java/com/baeldung/websocket/WebSocketController.java]: To have the file formatted similarly. Before each @Bean there is an empty line, so there should be also.

File: spring-reactive-modules/spring-5-reactive/src/main/java/com/baeldung/websocket/ReactiveWebSocketConfiguration.java (hunk 2 above)
Comment (andrzejsydor) [reply spring-reactive-modules/spring-5-reactive/src/main/java/com/baeldung/websocket/ReactiveWebSocketConfiguration.java:54]: To have the file formatted similarly. Before each @bean there is an empty line, so there should be also.

Comment (jsgrah-spring) [reply spring-reactive-modules/spring-5-reactive/src/main/java/com/baeldung/websocket/ReactiveWebSocketConfiguration.java:55]: Thanks @andrzejsydor . I have updated to add the new line.

**Pull request closed without merging**